nemo/collections/asr/parts/utils/transcribe_utils.py

```diff
@@ -227,10 +226,8 @@ def prepare_audio_data(cfg: DictConfig) -> Tuple[List[str], bool]:
                 else:
                     has_two_fields.append(False)
                 audio_key = cfg.get('audio_key', 'audio_filepath')
-                audio_file = Path(item[audio_key])
-                if not audio_file.is_file() and not audio_file.is_absolute():
```

Comment:
In reluctant to change any of this. It has severe negative interaction with tarred datasets and filename size limits.